pasystem/pasystem-tool/tool/src/webapp/scripts/pasystem-tool.js

```diff
@@ -23,7 +23,6 @@ $(function () {
             // add clear action if present
             $datepicker.siblings().find(".clear-datepicker-btn").click(function() {
               $datepicker.val("");
-              $("[name='" + $datepicker.attr('id') + "_selected_datetime']").val("");
```

Comment:
I wouldn't just remove this, unless you know exactly where it's used. It looks like it's the hidden field generated by the date picker. I'd play it safe and do:
const hidden = this.querySelector(`[name='${$datepicker.attr('id')}_selected_datetime']`);
hidden && hidden.value = "";